### PR TITLE
eth, internal/web3ext: admin flag to allow reenabling fast sync

### PR DESCRIPTION
**Pull request by karalabe**

### CODE REVIEW

File: eth/api.go [hunk 1]
```diff
@@ -275,6 +277,23 @@ func (api *PrivateAdminAPI) ImportChain(file string) (bool, error) {
 	return true, nil
 }
 
+// FastSync reenables fast sync for a shot shot sync round. Use with extreme caution.
+func (api *PrivateAdminAPI) FastSync() {
+	// Create a diplaimer warning message for the user
```
Comment (bas-vk) [inline eth/api.go]: `s/diplaimer/disclamer`

File: internal/web3ext/web3ext.go [hunk 2]
```diff
@@ -198,6 +198,11 @@ web3._extend({
 			params: 1
 		}),
 		new web3._extend.Method({
+			name: 'fastSync',
+			call: 'admin_fastSync',
+			params: 0
```
Comment (bas-vk) [inline internal/web3ext/web3ext.go]: It's possible to omit params, 0 is the default value.

**Comment by karalabe** [comment]
This wasn't ready for merge.

**Comment by fjl** [comment]
Hmm

I merged because it was approved and the comments were just aesthetics.

**Comment by karalabe** [comment]
I'll look into it deeper. There was an issue where the sync progress was not correctly reset.

**Comment by karalabe** [comment]
That's why I didn't remove the `progress` label, though I should have documented it here, so my fault too. Though nothing serious so it can sty merged.

**Pull request merged**